examples/tv-casting-app/tv-casting-common/core/CastingPlayer.cpp

```diff
@@ -243,6 +253,27 @@ CHIP_ERROR CastingPlayer::StopConnecting()
     mTargetCastingPlayer.reset();
     CastingPlayerDiscovery::GetInstance()->ClearCastingPlayersInternal();
 
+    // shouldSendIdentificationDeclarationMessage is false only when StopConnecting() is called by the casting library.
+    if (!shouldSendIdentificationDeclarationMessage)
+    {
+        ChipLogProgress(AppServer,
+                        "CastingPlayer::StopConnecting() shouldSendIdentificationDeclarationMessage: %s, User Directed "
+                        "Commissioning stopped by CastingPlayer/Commissioner user",
+                        shouldSendIdentificationDeclarationMessage ? "true" : "false");
```

Comment:
```suggestion
        ChipLogProgress(AppServer,
                        "CastingPlayer::StopConnecting() shouldSendIdentificationDeclarationMessage: %d, User Directed "
                        "Commissioning stopped by CastingPlayer/Commissioner user",
                        shouldSendIdentificationDeclarationMessage);
```

Reply:
That's fine, but it will be less readable in the logs.